packages/trader/src/App/Components/Routes/binary-link.tsx

```diff
@@ -3,39 +3,45 @@ import { NavLink } from 'react-router-dom';
 import { findRouteByPath, normalizePath } from './helpers';
 import getRoutesConfig from '../../Constants/routes-config';
 
-type TBinaryLinkProps = React.PropsWithChildren<{
-    active_class?: string;
-    className?: string;
-    to?: string;
-    onClick?: () => void;
-}>;
+type TBinaryLinkProps = Omit<React.HTMLProps<HTMLAnchorElement>, 'title' | 'ref'> &
+    React.PropsWithChildren<{
+        active_class?: string;
+        className?: string;
+        to?: string;
+        onClick?: () => void;
+    }>;
```

Comment:
Here we updated the props type in order to be able to pass some native event handlers like `onDragStart` and added `forwardRef` because we are using this component in `ContractCardComponent` with `react-swipeable` which passes a `ref` down to `BinaryLink`.